src/components/Home/Home.js

```diff
@@ -38,9 +38,12 @@ export default class Home extends React.Component {
   
   hidemovie = () => {
     this.state.effect.from('.home', 
-    { ease: Back.easeOut, x: 2990, duration: 1.5}).then(() => {
+    { ease: Back.easeOut, x: 2990, duration: 1.5})
+    // .then(() => {
+    //   document.querySelector('.home').removeAttribute('style')
+    // })
       document.querySelector('.home').removeAttribute('style')
```

Comment:
I remember trying this without the 'then()' and it failing, but if it works now, then I'm fine with this.